modules/distr-tracing-tempo-remove-cli.adoc

```diff
@@ -0,0 +1,52 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-tempo-removing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-removing-tempo-instance-cli_{context}"]
+= Removing a TempoStack instance by using the CLI
+
+You can remove a TempoStack instance on the command line.
+
+.Prerequisites
+
+* An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your OpenShift CLI (`oc`) version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+.Procedure
+
+. Get the name of the TempoStack instance:
++
+[source,terminal]
+----
+$ oc get deployments -n <openshift_project_of_tempostack_instance>
+----
+
+. Remove the TempoStack instance:
```

Comment:
CLI step needs `by running the following command:`

Reply:
Fixed.